sphinx_gherkindoc/utils.py

```diff
@@ -104,7 +104,8 @@ def add_output(self, line: str, line_breaks: int = 1, indent_by: int = 0) -> Non
             line_breaks: The number of line breaks to include
             indenty_by: The number of spaces to indent the line.
         """
-        self._output.append(u"{}{}{}".format(" " * indent_by, line, "\n" * line_breaks))
+        line_breaks_str = "\n" * line_breaks
+        self._output.append(f"{' ' * indent_by}{line}{line_breaks_str}")
```

Comment:
Nice, the intermediate variable makes for greater clarity even though it adds a line.